src/PayU/Alu/Response.php

```diff
@@ -376,6 +388,7 @@ public function parseAdditionalParameters($parameters)
             'ECI',
             'CAVV',
             'TRANSID',
+            'URL_REDIRECT',
```

Comment:
`parseAdditionalParameters` method is used for reading and parsing ALU response plugins. Let's keep it's scope to that and not add `URL_REDIRECT` here.